pkgs/top-level/all-packages.nix

```diff
@@ -1799,6 +1799,8 @@ with pkgs;
 
   systeroid = callPackage ../tools/system/systeroid { };
 
+  tailwindcss-bin = callPackage ../development/tools/tailwindcss { };
```

Comment:
We don't suffix binary packages with -bin

Reply:
When I search for `-bin` in nixpkgs I see many results with this suffix and the description "binary package". How are those different?

Reply:
we don't do these anymore since meta.sourceProvonenace got introduced and there is only one variant of the package.

Reply:
Ok, here's a PR to rename it: https://github.com/NixOS/nixpkgs/pull/241570